inst/include/h_util.hpp

```diff
@@ -28,6 +28,6 @@
  */
 #define MODEL_VERSION "2.5.0"
 
-#define OUTPUT_DIRECTORY "output/"
+#define OUTPUT_DIRECTORY "/Users/dorh012/Desktop/"
```

Comment:
Um we probably don't want this :)

Reply:
Is this when working in R? Output is definitely disabled in that case. Or a C++ side oddity?

Reply:
No it does not work on the R side of things.

Reply:
No, it shouldn't; the csvoutputstream visitor is disabled when running through R.

Reply:
Do we expect any output to be written when running from Xcode or executing from the command line?

Reply:
Yes